tests/ios/unit-test/test-imports-app/frameworks/Basic/BUILD.bazel

```diff
@@ -0,0 +1,13 @@
+load("//rules:framework.bzl", "apple_framework")
+
+apple_framework(
+    name = "Basic",
+    # Note: it is totally possible that a user would write a glob like this..
+    # srcs = glob([
+    #    "sources/Basic/**/*.h",
+    # ]),
```

Comment:
So the ideal is to keep this as a note for someone trying to understand what is under test right? Why is it relevant to know that such a `glob` could be written?

Reply:
Yeah it is possible a user to craft an redundant build file which duplicates `srcs` and `framework_imports` - probably better to have the comment but I can remove it doesn't make sense. The rule will handle it anyways.

Reply:
I'm ok with keeping it. Was just curious to know the details of the edge case you're mentioning here 👍